charts/netdata/values.yaml

```diff
@@ -330,3 +330,130 @@ child:
     token: ""
     rooms: ""
     url: "https://app.netdata.cloud"
+
+k8sState:
+  enabled: false
+  port: "{{ .Values.parent.port }}"
+
+  resources: {}
+    # limits:
+    #  cpu: 4
+    #  memory: 4096Mi
+    # requests:
+    #  cpu: 4
+    #  memory: 4096Mi
+
+  livenessProbe:
+    failureThreshold: 3
+    periodSeconds: 30
+    successThreshold: 1
+    timeoutSeconds: 1
+  readinessProbe:
```

Comment:
`readinessProbe` having the same values as `livenessProbe` make it useless

Reply:
I agree. In addition, the defaults we have are not good IMO. However, we have them for all Netdatas:
 - parent
 - children

So I just copied them. Changing the values looks like a separate PR for me.